### ISSUE TITLE
[CVE Report]: messagebird/sachet:0.2.3 

### ISSUE THREAD

**Issue by jiaqiluo** [messagebird/sachet#87]
Can you publish a new docker image to address those CVEs? 

|Vulnerability ID|Title|Package Name|Fixed Version|Severity|URL|
|---|---|---|---|---|---|
|CVE-2021-30139||apk-tools|2.10.6-r0|HIGH|https://avd.aquasec.com/nvd/cve-2021-30139|
|CVE-2021-28831|busybox: invalid free or segmentation fault via malformed gzip data|busybox|1.31.1-r10|HIGH|https://avd.aquasec.com/nvd/cve-2021-28831|
|CVE-2020-1967|openssl: Segmentation fault in SSL_check_chain causes denial of service|libcrypto1.1|1.1.1g-r0|HIGH|https://avd.aquasec.com/nvd/cve-2020-1967|
|CVE-2021-23840|openssl: integer overflow in CipherUpdate|libcrypto1.1|1.1.1j-r0|HIGH|https://avd.aquasec.com/nvd/cve-2021-23840|
|CVE-2021-3450|openssl: CA certificate check bypass with X509_V_FLAG_X509_STRICT|libcrypto1.1|1.1.1k-r0|HIGH|https://avd.aquasec.com/nvd/cve-2021-3450|
|CVE-2020-1967|openssl: Segmentation fault in SSL_check_chain causes denial of service|libssl1.1|1.1.1g-r0|HIGH|https://avd.aquasec.com/nvd/cve-2020-1967|
|CVE-2021-23840|openssl: integer overflow in CipherUpdate|libssl1.1|1.1.1j-r0|HIGH|https://avd.aquasec.com/nvd/cve-2021-23840|
|CVE-2021-3450|openssl: CA certificate check bypass with X509_V_FLAG_X509_STRICT|libssl1.1|1.1.1k-r0|HIGH|https://avd.aquasec.com/nvd/cve-2021-3450|
|CVE-2021-28831|busybox: invalid free or segmentation fault via malformed gzip data|ssl_client|1.31.1-r10|HIGH|https://avd.aquasec.com/nvd/cve-2021-28831|

**Comment by jiaqiluo** [comment]
bump

**Comment by marcelcorso** [comment]
hey @jiaqiluo ! Thank you for the heads up. Can you help me a little bit? I'm not sure how to fix these. 

For example: we don't use "busybox" on the container described on Dockerfile. 

And about the openssl CVEs: updating the `alpine` image should fix them?

**Comment by marcelcorso** [comment]
@jiaqiluo

https://hub.docker.com/layers/messagebird/sachet/0.2.6/images/sha256-df8da6ba98d09b179bdc0486629a96c2a4dd8c6f1dd9877f4b5b45afc3a327ac?context=explore

**Comment by jiaqiluo** [comment]
I can confirm all CVEs are fixed in messagebird/sachet:0.2.6
```
> trivy image messagebird/sachet:0.2.6
2021-08-30T09:49:33.192-0700	INFO	Need to update DB
2021-08-30T09:49:33.192-0700	INFO	Downloading DB...
23.09 MiB / 23.09 MiB [--------------------------------------------------------------------------------------------------------------------------------------] 100.00% 26.17 MiB p/s 2s
2021-08-30T09:49:38.262-0700	INFO	Detected OS: alpine
2021-08-30T09:49:38.263-0700	INFO	Detecting Alpine vulnerabilities...
2021-08-30T09:49:38.264-0700	INFO	Number of language-specific files: 0

messagebird/sachet:0.2.6 (alpine 3.14.2)
========================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
